src/openrct2/Cheats.h

```diff
@@ -11,6 +11,13 @@
 
 #include "common.h"
 
+typedef enum _WindowCheatsStaffSpeed
```

Comment:
Use enum class, also this has little to do with the window itsself. Use a name like `StaffSpeedCheat`